### PR TITLE
Adding missing animations on TripOverview in Booking Details

### PR DESCRIPTION
**Pull request by RobinCsl**
and removing visual feedback on Touchable

### CODE REVIEW

File: packages/shared/src/icons/TextIcon.js [hunk 1]
```diff
@@ -30,19 +30,20 @@ const styleSheet = StyleSheet.create({
...
+      fontSize: 20,
+    },
+  };
+  render() {
+    return (
+      <Text style={[styleSheet.icon, this.props.style]}>
+        <Translation passThrough={this.props.code} />
+      </Text>
+    );
+  }
```
Comment (tbergq) [inline packages/shared/src/icons/TextIcon.js]: Why is this change needed?

Comment (RobinCsl) [reply packages/shared/src/icons/TextIcon.js]: This is to be able to use `Animated.createAnimatedComponent`; it was complaining that it only works with classes.
